scripts/common/_configuration.ps1

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+function loadTestProjectList()
+{
+    return Import-CSV "$RepoRoot\scripts\configuration\testProjects.csv" -Header "ProjectName"
```

Comment:
I'm not particularly in love with it either, but I think it improves on what was previously there. 
The unix and Windows version were already way out of sync.